tools/strings-coverage.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+
+import sys
+import fileinput
+from babel import Locale
```

Comment:
Would we have to install this module when running the action? I tried to run the script on my Mac, and it failed this dependency.

Reply:
Good catch! It was already installed on my system so I thought it was part of the standard libs. I tested quickly on a vanilla Ubuntu and it wasn't installed. I will add it to the GH action.

Reply:
Happens to me all the time - with Perl dependencies 😁.